lms/lmsweb/views.py

```diff
@@ -208,6 +208,51 @@ def share():
     return fail(400, f'Unknown or unset act value "{act}".')
 
 
+@webapp.route('/notes', methods=['GET', 'POST'])
+@login_required
+def note():
+    if not current_user.role.is_manager:
+        return fail(403, "You aren't allowed to access this page.")
+
+    act = request.args.get('act') or request.json.get('act')
+    user_id = request.args.get('userId')
+
+    user = User.get_or_none(User.id == user_id)
+    if user is None:
+        return fail(404, f'No such user {user_id}.')
+
+    if act == 'delete':
+        note_id = int(request.args.get('noteId'))
+        note_ = Note.get_or_none(Note.id == note_id)
+        if (
+            note_.creator.id != current_user.id
+            and note_.is_private
+        ):
+            return fail(403, "You aren't allowed to access this page.")
+        if note_ is not None:
+            note_.delete_instance()
+        return jsonify({'success': 'true'})
+
+    if act == 'create':
+        note_text = request.form.get('note', '')
```

Comment:
Our endpoints shouldn't mess with forms, we should create a unified REST API.

Change it to POST request.